searchlib/src/vespa/searchlib/memoryindex/document_inverter_collection.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "document_inverter_collection.h"
+#include "document_inverter.h"
+#include <cassert>
+
+namespace search::memoryindex {
+
+DocumentInverterCollection::DocumentInverterCollection(DocumentInverterContext& context)
+    : _context(context),
+      _free_inverters(),
+      _inflight_inverters(),
+      _active_inverter(std::make_unique<DocumentInverter>(_context)),
+      _num_inverters(1),
+      _max_inverters(4)
```

Comment:
Consider adding this as a constructor argument.

Reply:
Done.